### PR TITLE
documents referring to ns1labs

### PR DESCRIPTION
**Pull request by manrodrigues**
this PR is not to be merged.. just to make easy to track files that may need to be changed

### CODE REVIEW

**Comment by manrodrigues** [comment]
> @manrodrigues Any reason not to merge this now?

@rboucher-me 
Actually, here in Orb this PR is only a high level to make easier to check all the files with reference to ns1labs.. and validate that everton's changes + the UI ones are enough and won't break anything. I didn't test anything... just replaced everything. That's why I don't want to merge it now.

File: .github/workflows/go-production.yml [hunk 1]
```diff
@@ -186,7 +186,7 @@ jobs:
       - name: Build orb-agent
         shell: bash
         env:
-          IMAGE_NAME: ns1labs/orb-agent
+          IMAGE_NAME: orbcommunity/orb-agent
```
Comment (gpazuch) [inline .github/workflows/go-production.yml:189]: just checking - is this supposed to be `orbcommunity` or `orb-community`?

**Review by gpazuch — APPROVED** [review]
I left a couple of comments just to double check the intended changes.

File: .github/workflows/go-production.yml (hunk 1 above)
Comment (manrodrigues) [reply .github/workflows/go-production.yml:189]: it's orbcommunity for docker hub and orb-community for github :)

File: ui/src/app/@theme/components/footer/footer.component.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <span class="created-by">
   <p>  
-    Made with ♥ by <b><a href="https://ns1.com/labs.com" target="_blank">NS1Labs</a></b> 2021
+    Made with ♥ by <b><a href="https://netboxlabs.com/home" target="_blank">orbcommunity</a></b> 2021
```
[the remark this answers is not in the document]
Comment (manrodrigues) [reply ui/src/app/@theme/components/footer/footer.component.html]: Check now, please

**Review by gpazuch — APPROVED** [review]

**Pull request merged**